build.psm1

```diff
@@ -158,7 +158,30 @@ function Start-PSBuild {
     }
 
     # save Git description to file for PowerShell to include in PSVersionTable
-    git --git-dir="$PSScriptRoot/.git" describe --dirty --abbrev=60 > "$psscriptroot/powershell.version"
+    $powershellVersion = git --git-dir="$PSScriptRoot/.git" describe --dirty --abbrev=60
+    $powershellVersion > "$PSScriptRoot/powershell.version"
```

Comment:
I use the file in a test. Can we leave the file in repo but exclude it from release and publish?